include/zephyr/bluetooth/conn.h

```diff
@@ -1588,14 +1588,14 @@ enum bt_conn_le_cs_procedure_enable_state {
  *  - 2:2 configuration, where both A and B support 2 antennas and N_AP = 4
  */
 enum bt_conn_le_cs_tone_antenna_config_selection {
-	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_INDEX_ONE = BT_HCI_OP_LE_CS_ACI_0,
-	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_INDEX_TWO = BT_HCI_OP_LE_CS_ACI_1,
-	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_INDEX_THREE = BT_HCI_OP_LE_CS_ACI_2,
-	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_INDEX_FOUR = BT_HCI_OP_LE_CS_ACI_3,
-	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_INDEX_FIVE = BT_HCI_OP_LE_CS_ACI_4,
-	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_INDEX_SIX = BT_HCI_OP_LE_CS_ACI_5,
-	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_INDEX_SEVEN = BT_HCI_OP_LE_CS_ACI_6,
-	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_INDEX_EIGHT = BT_HCI_OP_LE_CS_ACI_7,
+	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_A1_B1 = BT_HCI_OP_LE_CS_ACI_0,
+	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_A2_B1 = BT_HCI_OP_LE_CS_ACI_1,
+	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_A3_B1 = BT_HCI_OP_LE_CS_ACI_2,
+	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_A4_B1 = BT_HCI_OP_LE_CS_ACI_3,
+	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_A1_B2 = BT_HCI_OP_LE_CS_ACI_4,
+	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_A1_B3 = BT_HCI_OP_LE_CS_ACI_5,
+	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_A1_B4 = BT_HCI_OP_LE_CS_ACI_6,
+	BT_LE_CS_TONE_ANTENNA_CONFIGURATION_A2_B2 = BT_HCI_OP_LE_CS_ACI_7,
```

Comment:
I see _ANT_  and _CONF_  used in Bluetooth public nterfaces; would it make sense for CS feature too use the same? or keep the _ANTENNA_ (I see DF has used `BT_DF_ANTENNA_*`) but change to use `_CONF_` for CS?

```suggestion
	BT_LE_CS_TONE_ANT_CONF_00_A1_B1 = BT_HCI_OP_LE_CS_ACI_0,
	BT_LE_CS_TONE_ANT_CONF_01_A2_B1 = BT_HCI_OP_LE_CS_ACI_1,
	BT_LE_CS_TONE_ANT_CONF_02_A3_B1 = BT_HCI_OP_LE_CS_ACI_2,
	BT_LE_CS_TONE_ANT_CONF_03_A4_B1 = BT_HCI_OP_LE_CS_ACI_3,
	BT_LE_CS_TONE_ANT_CONF_04_A1_B2 = BT_HCI_OP_LE_CS_ACI_4,
	BT_LE_CS_TONE_ANT_CONF_05_A1_B3 = BT_HCI_OP_LE_CS_ACI_5,
	BT_LE_CS_TONE_ANT_CONF_06_A1_B4 = BT_HCI_OP_LE_CS_ACI_6,
	BT_LE_CS_TONE_ANT_CONF_07_A2_B2 = BT_HCI_OP_LE_CS_ACI_7,
```